eng/common/scripts/Package-Properties.ps1

```diff
@@ -130,6 +130,17 @@ function Get-PrPkgProperties([string]$InputDiffJson) {
         {
             $filePath = Resolve-Path (Join-Path $RepoRoot $file)
             $shouldInclude = $filePath -like "$pkgDirectory*"
+            $pathComponents = $file -split "/"
+
+            # sdk/<service>/<file.extension>
+            if ($pathComponents.Length -eq 3 -and $pathComponents[0] -eq "sdk") {
```

Comment:
While this works it might end up pulling in more packages then necessary. Consider likes like https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/eventhub or https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/storage where the service directory is split up into different pieces. Perhaps it isn't that big of a deal but something we should at least think through.

Reply:
Your concerns are valid. I'm just going to do languages-specific additions to `AdditionalValidationPackagesFromPackageSetFn `